### PR TITLE
add cronjob for update ranking

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #151** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/151" target="_blank">Graphite</a> 
    * **PR #152** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/152" target="_blank">Graphite</a> 
      * **PR #154** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/154" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/154?utm_source=stack-comment).

File: infra/helm/scc-server/templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: scc-update-ranking
```
Comment (Zeniuus) [inline infra/helm/scc-server/templates/cronjob.yaml:4]: 파일명도 scc-update-ranking.yaml이면 더 좋을 듯!

File: infra/helm/scc-server/templates/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      template:
+        spec:
+          containers:
+          - name: curl-container
+            image: appropriate/curl
+            args: 
+            - "--fail"
+            - "-XPOST"
+            - "http://scc-server.{{ .Release.Namespace }}.svc.cluster.local/updateAccessibilityRanks"
+          restartPolicy: OnFailure
```
Comment (Zeniuus) [inline infra/helm/scc-server/templates/cronjob.yaml:20]: 무제한으로 재시도하나요? -> [6회가 default군요](https://kubernetes.io/docs/concepts/workloads/controllers/job/#pod-backoff-failure-policy)

**Review by Zeniuus — APPROVED** [review]

File: infra/helm/scc-server/templates/cronjob.yaml (hunk 1 above)
Comment (bellatoris) [reply infra/helm/scc-server/templates/cronjob.yaml:4]: 여기다가 `---` 를 이용해서 계속 붙일생각이었습니다.

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/154).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/154) failed to merge this pull request for an unknown reason. Try your merge again, or report a bug if you see this consistently.

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/154).

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/154).

**Pull request merged**